spec.html

```diff
@@ -762,7 +763,7 @@ <h1>Grammar Notation</h1>
           DecimalDigit [lookahead &lt;! DecimalDigit]
       </emu-grammar>
       <p>matches either the letter `n` followed by one or more decimal digits the first of which is even, or a decimal digit not followed by another decimal digit.</p>
-      <p>Similarly, if the phrase &ldquo;[lookahead &isin; _set_]&rdquo; appears in the right-hand side of a production, it indicates that the production may only be used if the immediately following input token sequence is a member of the given _set_. If the _set_ consists of a single terminal the phrase &ldquo;[lookahead = _terminal_]&rdquo; may be used.</p>
+      <p>Note that when these phrases are used in the syntactic grammar it may not be possible to unambiguously identify the immediately following token sequence, because determining later tokens requires knowing which lexical goal symbol to use at later positions. As such, when these are used in the syntatic grammar each token sequence _seq_ must be such that the choices of lexical goal symbols to use when scanning the following input text cannot change whether or not _seq_ is a prefix of the resulting token sequence.</p>
```

Comment:
```suggestion
      <p>Note that when these phrases are used in the syntactic grammar it may not be possible to unambiguously identify the immediately following token sequence, because determining later tokens requires knowing which lexical goal symbol to use at later positions. As such, when these are used in the syntactic grammar, each token sequence _seq_ is such that the choices of lexical goal symbols to use when scanning the following input text cannot change whether or not _seq_ is a prefix of the resulting token sequence.</p>
```
("must be" sounds like a requirement on implementations, whereas this is stating a self-imposed constraint on the spec.)